src/lib/y2storage/dialogs/guided_setup/select_scheme.rb

```diff
@@ -141,11 +146,25 @@ def password_correct?
         end
 
         # Password is considered strong when cracklib returns an empty message.
+        # User has the last word to decide whether to use a weak password.
         def password_strong?
           message = cracklib_message
           return true if message.empty?
-          Yast::Report.Warning(message)
-          false
+
+          popup_text = [
+            _("The password is too simple:"),
```

Comment:
well, in such case we should prefer consistency of yast and installation. Even when my proposal is more consistent with the most common web ui wording.